Dockerfile

```diff
@@ -41,6 +41,8 @@ RUN chmod +x /okteto/bin/okteto
 # Test
 RUN /okteto/bin/okteto version
 
+COPY docker-credential-okteto /okteto/bin/docker-credential-okteto
```

Comment:
is it possible to make it work in the installer too?
That way people could use third party tooling when not using `--remote?

Reply:
I just opened this PR: https://github.com/okteto/pipeline-installer-image/pull/114. Is that what you mean?

Reply:
Exactly, thanks @maroshii !